SchibstedAccount/UI/Coordinators/UpdatedTermsCoordinator.swift

```diff
@@ -0,0 +1,124 @@
+//
+// Copyright 2011 - 2018 Schibsted Products & Technology AS.
+// Licensed under the terms of the MIT license. See LICENSE in the project root.
+//
+
+import UIKit
+
+class UpdatedTermsCoordinator: FlowCoordinator {
```

Comment:
Can we not use the existing CompleteProfileCoordinator? - it will show the screen and then it just won't do any of the required fields entry since there won't be any (if there are then it's also probably the correct thing to do with gdpr, etc, anyway).

Reply:
I was thinking that ideally `CompleteProfileCoordinator` should have used its own instance of `UpdatedTermsCoordinator` in a subsequent refactoring, but if it's a good thing to check for the required fields as well then probably `CompleteProfileCoordinator` can be adapted to work in this case as well.

Reply:
Ahh, yeah ok. I'm not sure if it's a good thing anymore actually :p